facilitator/src/bin/facilitator.rs

```diff
@@ -539,6 +542,50 @@ fn main() -> Result<(), anyhow::Error> {
                 .add_packet_decryption_key_argument()
                 .add_batch_signing_key_arguments()
         )
+        .subcommand(
+            SubCommand::with_name("lint-manifest")
+            .about("Validate and print out global or specific manifests")
+            .arg(
+                Arg::with_name("manifest-base-url")
+                    .long("manifest-base-url")
+                    .value_name("URL")
+                    .help("base URL relative to which manifests may be fetched")
+                    .long_help(
+                        "base URL relative to which manifests may be fetched \
+                        over HTTPS. Should be in the form \"https://foo.com\"."
+                    )
+                    .required_unless("manifest-path")
+            )
+            .arg(
+                Arg::with_name("manifest-path")
+                    .long("manifest-path")
+                    .value_name("PATH")
+                    .help("path to local manifest file to lint")
+                    .required_unless("manifest-base-url")
+            )
+            .arg(
+                Arg::with_name("manifest-kind")
+                    .long("manifest-kind")
+                    .value_name("KIND")
+                    .help("kind of manifest to locate and parse")
+                    .possible_value(leak_string(ManifestKind::IngestorGlobal.to_string()))
+                    .possible_value(leak_string(ManifestKind::DataShareProcessorGlobal.to_string()))
+                    .possible_value(leak_string(ManifestKind::DataShareProcessorSpecific.to_string()))
+                    .possible_value(leak_string(ManifestKind::PortalServerGlobal.to_string()))
+                    .required(true)
+                )
+            .arg(
+                Arg::with_name("instance")
+                    .long("instance")
+                    .value_name("LOCALITY-TECH_GIANT")
```

Comment:
For consistency let's say:

```suggestion
                    .value_name("LOCALITY-INGESTOR")
```